src/utils/pickupLocationsUtils.ts

```diff
@@ -0,0 +1,50 @@
+import { type SierraCodeName } from "../types/myAccountTypes"
+
+const cache = {
+  pickupLocations: null,
+  lastUpdated: null,
+  isStillValid: function () {
+    const now = Date.now()
+    return this.lastUpdated && now - this.lastUpdated < 360000
+  },
+}
+
+export const testUtils = {
```

Comment:
If this is just being used for tests can we export the functions and move this to the test file?

Reply:
I'm using these methods to verify and reset the cache, so it has to be from this file.